### DISCUSSION TITLE
What is Distributed version control system

### DISCUSSION THREAD

**Discussion by avaniishh123** [community/community#84972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please answer

**Comment by vivekvardhan2810** [comment]
A distributed [version control system](https://about.gitlab.com/topics/version-control/) (DVCS) brings a local copy of the complete repository to every team member’s computer, so they can commit, branch, and merge locally. I hope you got an answer

**Comment by avaniishh123** [comment]
Thank You
